components/formats-api/src/loci/formats/in/DynamicMetadataOptions.java

```diff
@@ -30,26 +30,71 @@
  * #L%
  */
 
-package loci.formats;
+package loci.formats.in;
 
 import java.util.Properties;
 import java.io.File;
 
 
 /**
- * Generic configuration object for readers and writers.
+ * Configuration object for readers and writers.
```

Comment:
Once reviewed, it might be worth expanding on the description of this new implementation and its scope of usage.